bot/cogs/logging.py

```diff
@@ -15,8 +17,26 @@ def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
     async def on_ready(self):
+        log.info('Signed in as:')
+        log.info('--------------')
+        log.info(f'Username: {self.bot.user.name}')
+        log.info(f'User ID: {self.bot.user.id}')
+        log.info('--------------')
+        log.info('Serving Team 17 in Code Jam 1!')
+        log.info('--------------')
         log.info("Bot connected!")
 
+        self.bot.session = ClientSession(loop=self.bot.loop)
+        self.bot.info_url = 'https://snake-facts.weebly.com/'
+        log.info('Session created!')
+
+        with open('./snakes.txt', encoding='utf-8') as f:
+            self.bot.sneks = f.read().split('\n')
+            for i, snek in enumerate(self.bot.sneks):
+                self.bot.sneks[i] = snek.replace('\u200b', '').replace('\ufeff', '')
+
+        log.info('Snakes loaded.')
```

Comment:
This is the logging cog. If you need to do stuff `on_ready`, do it in your cog.

Reply:
you can make two `on_ready` events in two different files?

Reply:
You sure can!

Reply:
what order will the bot perform the actions in?

Reply:
No idea. You shouldn't rely on that.